Updates to trademark survey questions

joshtriplett opened the pull request:
Various updates to the questions and introduction of the trademark survey,
attempting to gather data that will best inform policy development.

May be best reviewed commit-by-commit; each commit is one change.

This is net neutral on number of questions asked; it does not make the survey any longer.

JoelMarcey commented on surveys/2021-trademark-survey/questions.md:
```diff
@@ -50,9 +51,9 @@ are used for. Trademark law also covers confusion about whether the trademark
 owner is endorsing or sponsoring the user. For example, when websites include
 their customers’ logos on their websites, we understand that the customer’s
 logo is being used to signal that the owner of the logo approves of or endorses
```
```suggestion
logo is being used to signal that the owner of that logo approves of or endorses
```

JoelMarcey commented on surveys/2021-trademark-survey/questions.md:
```diff
@@ -50,9 +51,9 @@ are used for. Trademark law also covers confusion about whether the trademark
-the product, and we take that information into account when considering the
-product. A suggestion of endorsement that is inaccurate can also be a trademark
-infringement.
+the product and possibly the content of the website, and we take that
```
```suggestion
the owner of the website, and we take that
```

JoelMarcey commented on surveys/2021-trademark-survey/questions.md:
```diff
@@ -50,9 +51,9 @@ are used for. Trademark law also covers confusion about whether the trademark
-product. A suggestion of endorsement that is inaccurate can also be a trademark
-infringement.
+the product and possibly the content of the website, and we take that
+information into account when considering the product and website. A suggestion
```
```suggestion
information into account when considering the owner of the website. A suggestion
```

JoelMarcey replied:
These two sentences read odd to me and this is my attempt to maybe make it read less odd.

joshtriplett replied:
Seems reasonable to me, in the same spirit; any of those endorsements could be possible.

JoelMarcey commented on surveys/2021-trademark-survey/questions.md:
```diff
@@ -148,7 +150,7 @@ Pick as many as you agree with:
 > TODO
 
-### Commercial businesses can use the Rust/Cargo logo on their website or social media site, provided they have made a financial contribution to the Rust Foundation.
+### Commercial businesses can use the Rust/Cargo logo on their website or social media site, in the context of an open source project that uses Rust.
```
This change definitely makes the question different than the original intent. Not choosing sides on which is better or not, but I can actually see these potentially being two separate questions.

joshtriplett replied:
That's true. We could ask the other question ("provided they have an open source project that uses Rust"), but that seems like a much broader ask likely to get different answers. (And I was trying to avoid increasing the number of questions.)

joshtriplett replied:
Also, for context, note that GitHub's diff here is comparing to the wrong question here. The actual change is from "Commercial businesses can use the Rust/Cargo logo on their website or social media site, provided they have an open source project that uses Rust" to "Commercial businesses can use the Rust/Cargo logo on their website or social media site, in the context of an open source project that uses Rust"

rylev replied:
I think we're missing an "only" or "provided":

> Commercial businesses can use the Rust/Cargo logo on their website or social media site, *only* in the context of an open source project that uses Rust.

JoelMarcey reviewed (changes requested):
Thanks for the suggestions. I am still reviewing the proposed changes a bit, but wanted to get what I have out there now as I need to run to something else.

rylev commented on surveys/2021-trademark-survey/questions.md:
```diff
@@ -111,13 +112,14 @@ trademarks that, without the license, would be an infringing use.
 > TODO
 
-### How much of the Rust language can someone change and still call it “Rust”?
+### How much of the Rust language/toolchain can someone change and still call it “Rust”?
```
This question somewhat implicitly assumes a fork of the existing Rust toolchain. In other words, this seems to care about what the diff is between the existing Rust language and tooling and new artifacts that come from the existing ones. I do wonder if we're missing out of subtle distinctions when someone creates an entirely new artifact and calls it Rust. I think this distinction gets at the heart of the fact that we have two groupings of things that are called "Rust": the abstract language definition which isn't really a thing but more an idea, and the actual artifacts that implement that idea (e.g., the compiler, cargo, etc).

JoelMarcey replied:
I was trying to think of ways to reword this question to incorporate what I understand you to be getting at here, but it was tough given the responses we have for this question. I feel it would have to be new question of something like:

> When someone creates a new artifact for the Rust ecosystem, what is required to actually call it Rust?

The hard part here is coming up with the right choices for the survey responses.

rylev commented on surveys/2021-trademark-survey/questions.md:
```diff
@@ -224,9 +226,12 @@ Pick as many as you agree with:
 - No
 - Yes, but only if it’s used for something that has nothing to do with software
-- Yes, and it can be used even for software
+- Yes, and the license must qualify as Open Source
```
I'm not sure if everyone will have the same understanding of what "qualify as Open Source" means. Can we better qualify this?

joshtriplett commented:
On Mon, Aug 01, 2022 at 06:58:28AM -0700, Ryan Levick wrote:
> @rylev commented on this pull request.
> > @@ -111,13 +112,14 @@ trademarks that, without the license, would be an infringing use.
>  >
>  > TODO
>  
> -### How much of the Rust language can someone change and still call it “Rust”?
> +### How much of the Rust language/toolchain can someone change and still call it “Rust”?
> 
> This question somewhat implicitly assumes a fork of the existing Rust toolchain. In other words, this seems to care about what the diff is between the existing Rust language and tooling and new artifacts that come from the existing ones. I do wonder if we're missing out of subtle distinctions when someone creates an entirely new artifact and calls it Rust. I think this distinction gets at the heart of the fact that we have two groupings of things that are called "Rust": the abstract language definition which isn't really a thing but more an idea, and the actual artifacts that implement that idea (e.g., the compiler, cargo, etc). 

Valid, but I think a level of precision needed to make those fine
distinctions would make this survey too verbose with too many questions.

I changed it from "the Rust language/toolchain" to just "Rust". My goal
with this change was to make sure that we didn't *just* get information
about the Rust language, and not any sentiments regarding other aspects
(e.g. libraries, tooling, other associated tools).

Potential requirements for using the name in completely separate
software not derived from the Rust project would be sufficiently complex
that I don't think we can or should attempt to capture them in this
survey.

> >  - No
>  - Yes, but only if it’s used for something that has nothing to do with software
> -- Yes, and it can be used even for software
> +- Yes, and the license must qualify as Open Source
> 
> I'm not sure if everyone will have the same understanding of what "qualify as Open Source" means. Can we better qualify this?

I changed it to "must use an Open Source license", which should be
unambiguous.

> > @@ -148,7 +150,7 @@ Pick as many as you agree with:
>  >
>  > TODO
>  
> -### Commercial businesses can use the Rust/Cargo logo on their website or social media site, provided they have made a financial contribution to the Rust Foundation.
> +### Commercial businesses can use the Rust/Cargo logo on their website or social media site, in the context of an open source project that uses Rust.
> 
> I think we're missing an "only" or "provided":
> 
> > Commercial businesses can use the Rust/Cargo logo on their website or social media site, *only* in the context of an open source project that uses Rust.

Done; that does seem clearer.

JoelMarcey commented:
Thank you @joshtriplett and @rylev  for the suggestions and feedback. We have incorporated most, if not nearly all, of them. Here is the final survey that we are going be sending out, if you want to update this pull request for posterity...

```
Whether or not you use [Rust](https://rust-lang.org) today, we want to hear from
you!

The Rust Foundation has created this survey to help guide the development of a Rust trademark policy that best serves the users of the Rust programming language. The purpose of this questionnaire is to get directional information. For that reason, if you largely agree with an answer but not entirely, you should say you agree. There will be further refinements as the trademark guidelines are developed where we will be working out the nuances.

We are referring to the words RUST and CARGO and the logos, so assume the question is about both the words and the logos. If the question is about only the logos, it will be called “the Rust/Cargo logo.” Please note that the Rust logo only refers to the "gear" logo, not Ferris the Crab.

Unless you choose to enter your email, your answers will be anonymous. Any personal data you submit as a part of this survey will be handled in accordance with the [Rust Foundation Privacy Policy](https://foundation.rust-lang.org/policies/privacy-policy/).

We estimate it will take about 5-10 minutes to complete.

## Basic principles

Trademarks protect users from confusion, meaning that when they engage with a “Rust”- branded product or service they will be getting something with the qualities they expect. Someone will choose the same fast food restaurant in many different places because the brand signals to them that they will be getting food with the qualities and characteristics that they expect.

Trademarks for open source software have a wrinkle not found in other types of products. The copyright license allows the “product” – the software – to be changed by others. The question becomes at what point users will be misled if something is called “Rust” but has different qualities and characteristics than what is available from the original source.

This source identifying/quality assurance function is not all that trademarks are used for. Trademark law also covers confusion about whether the trademark owner is endorsing or sponsoring the user. For example, when customers’ logos are included on websites, we understand that the customer’s logo is being used to signal that the owner of that logo approves of or endorses the owner of the website, and we take that information into account when considering the owner of the website. A suggestion of endorsement that is inaccurate can also be a trademark infringement.

Trademark law always allows others to use a trademark to truthfully describe the product or service itself, such as “this program is written in the Rust language.” The Rust trademark policy would not try to stop this type of lawful use of the Rust trademark, or any lawful use of the Rust trademark.

With that background, a trademark policy is designed to do two things: (1) define what, in Rust’s view, is permitted, lawful use of their trademark and (2) optionally, grant some limited licenses allowing the use of the RUST trademarks that, without the license, would be an infringing use.

## Trademark Survey

### The Rust Foundation should be highly protectionist about trademarks.

- strongly agree
- agree
- neither agree nor disagree
- disagree
- strongly disagree

### Trademarks are different from copyrights, so it’s ok to exert some control or standards about how people use trademarks.

- strongly agree
- agree
- neither agree nor disagree
- disagree
- strongly disagree


### The Rust Foundation should share trademarks as freely as it shares code.

- strongly agree
- agree
- neither agree nor disagree
- disagree
- strongly disagree


### If someone creates a Rust package from the original source code without changing the code in any way, can they still call the compiled package “Rust”?

- Yes
- No


### How much of Rust can someone change and still call it “Rust”?

Pick as many as you agree with:

- Not at all
- Backporting changes already made or in progress in the Rust project is fine
- Fixed bugs and other minor changes are ok
- Extending with new functions/syntax but not removing anything or breaking anything
- They can change it any way they want

### Community members can put the Rust/Cargo logo on their personal website or blog.

- strongly agree
- agree
- neither agree nor disagree
- disagree
- strongly disagree

### Community members can use the Rust/Cargo logo as a profile picture on social media.

- strongly agree
- agree
- neither agree nor disagree
- disagree
- strongly disagree

### Commercial businesses can use the Rust/Cargo logo on their website or social media site, provided they have made a financial contribution to the Rust Foundation.

- strongly agree
- agree
- neither agree nor disagree
- disagree
- strongly disagree

### Commercial businesses can use the Rust/Cargo logo on their website or social media site, only in the context of an open source project that uses Rust.

- strongly agree
- agree
- neither agree nor disagree
- disagree
- strongly disagree

### Anyone can use the Rust/Cargo logo on their website or social media site for any reason, as long as they link to the Rust homepage.

- strongly agree
- agree
- neither agree nor disagree
- disagree
- strongly disagree

### Anyone can use the Rust/Cargo logo on their website or social media site, but if they use it, they must say they aren’t endorsed by the Rust Foundation.

- strongly agree
- agree
- neither agree nor disagree
- disagree
- strongly disagree

### Communities (e.g. discussion or chat sites) using the Rust trademarks must meet some minimum standards.

- strongly agree
- agree
- neither agree nor disagree
- disagree
- strongly disagree

### Can meetups, conferences, or other groups use the Rust/Cargo logos?

- Yes, anyone can, commercial or non-commercial
- Yes, only if they are non-commercial
- Yes, but only if the event follows standards the Rust Foundation has defined
- No

### Can others create promotional goods, like T-shirts, USB drives, stickers, socks, and hats with the Rust/Cargo logo? (Pick as many as you agree with)

- No, the Rust Foundation should be responsible for reviewing and approving all of these products
- No, the Rust Foundation should be responsible for ordering all of these products and handing them out
- Yes, for some types of items but not all kinds of items
- Yes, but they can only use designs that the Rust Foundation provides
- Yes, but only to give away, not for sale
- Yes, but only if some or all of the profits are donated to the Rust Foundation
- Yes, they can make whatever they like and keep the money

### The Rust/Cargo logo should be under a copyright license that allows modification and sharing. (Note: This does not either require or prevent the use of a trademark license with other requirements.)

- No
- Yes, but only if it’s used for something that has nothing to do with software
- Yes, and the license must be an Open Source license, such as MIT, Apache or the like.

### Please use this space to raise any other points you would like us to consider in reviewing our trademark policy.
```

joshtriplett commented:
Pushed a commit making the final version match that. Let's merge that version so that what's in the repo matches the survey that went out.
